chart/compass/charts/director/templates/ext-svc-certificate-rotation-job.yaml

```diff
@@ -1,153 +0,0 @@
-# This rotation job and all configurations related to it are temporary!
-apiVersion: batch/v1
-kind: CronJob
-metadata:
-  name: "{{ $.Chart.Name }}-ext-svc-certificate-rotation-job"
-  namespace: {{ $.Release.Namespace }}
-spec:
-  schedule: "{{ .Values.global.extSvcCertConfiguration.rotationCronjob.schedule }}"
-  concurrencyPolicy: Forbid
-  successfulJobsHistoryLimit: 5
-  failedJobsHistoryLimit: 5
-  startingDeadlineSeconds: 60
-  jobTemplate:
-    metadata:
-      labels:
-        cronjob: {{ .Values.global.extSvcCertConfiguration.rotationCronjob.name }}
-    spec:
-      template:
-        metadata:
-          labels:
-            cronjob: {{ .Values.global.extSvcCertConfiguration.rotationCronjob.name }}
-          annotations:
-            cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
-        spec:
-          serviceAccountName: {{ $.Chart.Name }}-{{ .Values.global.extSvcCertConfiguration.rotationCronjob.name }}
-          restartPolicy: Never
-          containers:
-            - name: {{ .Values.global.extSvcCertConfiguration.rotationCronjob.containerName }}
-              image: bitnami/kubectl:1.22.4
-              imagePullPolicy: IfNotPresent
-              command:
-                - /bin/bash
-                - -c
-                - |
-{{ .Files.Get "files/certificate_rotation.sh" | printf "%s" | indent 18 }}
-              env:
-                - name: EXT_NOT_EXTERNAL
```

Comment:
`EXT_NOT_EXTERNAL` becomes redundant with the removal of this job.

So, we can remove the if and leave what's in the else in the [certificate_rotation.sh](https://github.com/kyma-incubator/compass/blob/main/chart/compass/charts/director/files/certificate_rotation.sh#L75).

In addition, we can delete all occurrences of this env var from the charts where `certificate_rotation.sh` is invoked